RileyLinkKit/UI/RileyLinkDeviceTableViewController.swift

```diff
@@ -379,14 +432,22 @@ public class RileyLinkDeviceTableViewController: UITableViewController {
                 }
             case .PressDownButton:
                 vc = CommandResponseViewController { [unowned self] (completionHandler) -> String in
-                    self.device.ops?.pressButton({ (response) in
-                        dispatch_async(dispatch_get_main_queue()) {
-                            switch response {
-                            case .Success(let msg):
-                                completionHandler(responseText: "Result: " + msg)
-                            case .Failure(let error):
-                                completionHandler(responseText: String(error))
-                            }
+//                    self.device.ops?.pressButton({ (response) in
```

Comment:
ah, sorry about that. settings sync is something I hope to consider tackling soon.